frontend/src/components/sessions.js

```diff
@@ -45,3 +91,10 @@ SessionsList.propTypes = {
         PropTypes.shape({ Header: PropTypes.any.isRequired })
     ),
 };
+
+export const ConnectedSessionsList = connect(
+    (state) => ({
+        sessions: sessionsSelector(state),
```

Comment:
Let's switch this to the `useReducer` and `useDispatch` style.

Reply:
Do you mean `useSelector` from redux hooks or `useReducer` from React hooks?

Reply:
From `Redux`